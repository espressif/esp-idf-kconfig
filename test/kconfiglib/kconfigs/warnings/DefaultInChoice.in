mainmenu "Default value in choice config"

    choice CHOICE_NAME
        prompt "Choice prompt"

        config C1
            bool "C1"
            default y

    endchoice
