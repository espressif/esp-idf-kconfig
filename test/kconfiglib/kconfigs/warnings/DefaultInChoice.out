#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#

CONFIG_C1=y
